### DISCUSSION TITLE
购买的订阅提供了配置文件，命令行运行时，如何对个别配置项进行自行配置（覆盖）？

### DISCUSSION THREAD

**Discussion by TurnOffNOD** [MetaCubeX/mihomo#1180] [Q&A]
环境是在debian arm64，通过下载deb安装的mihomo。

现在有了`/etc/mihomo`文件夹，下面有个`config.yaml`，我自己购买的订阅是可以直接给完整的yaml配置文件的，现在也下载放到了`/etc/mihomo`文件夹下，命名为`my-sub.yaml`。

购买的订阅有些配置项（包括但不限于`allow-lan`、`fake-ip-filter`等等配置项）想自己设定新的值，但为了之后的更新订阅方便又不好直接修改`my-sub.yaml`的内容（否则下次更新就把自己的修改给覆盖了），想用`config.yaml`里面的对应项进行覆盖。

这个的命令行参数应该怎么用呢？或者有其它方法么？

**Comment by jaleel-liu** [comment]
https://wiki.metacubex.one/example/conf/
https://wiki.metacubex.one/config/proxy-providers/

**Reply by TurnOffNOD** [reply]
这些文档没有说明配置文件之间的组织结构。

**Comment by ZACKLZ** [comment]
同问楼主目前是否解决了问题，我也被此困扰= =

**Reply by mqcycy** [reply]
试试Openwrt-Nikki

**Reply by ZACKLZ** [reply]
谢谢，不过我用的docker版= =

**Comment by ignimutos** [comment]
把订阅的链接用作[代理集合](https://wiki.metacubex.one/config/proxy-providers/)，这样订阅就只会取节点数据，其他的都是由你自定义

**Reply by ZACKLZ** [reply]
感谢，已经成功配置
